packages/rxjs-firebase/src/operators/startWithDocumentSnapshotLoadingState.ts

```
@@ -0,0 +1,30 @@
+import { type DocumentData } from '@firebase/firestore'
```

ESLint

[error] Filename is not in kebab case. Rename it to `start-with-document-snapshot-loading-state.ts`.

(unicorn/filename-case)